File: Data Structures & Algorithms/binary-search/submission-0.cs
public class Solution {
    public int Search(int[] nums, int target) 
    {
        int l = 0;
        int h = nums.Length - 1;
        while(l <= h)
        {
            int m = (l + h) / 2;
            if(nums[m] == target)
            {
                return m;
            }
            else if(nums[m] > target)
            {
                h = m - 1;
            }
            else
            {
                l = m + 1;
            }
        }
        return -1;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) 
    {
        int maxconsecutive = 0;
        int consecutive = 0;
        for(int i = 0; i < nums.length; i++)
        {
            if(nums[i] == 1)
            {
                consecutive++;
                maxconsecutive = Math.max(consecutive, maxconsecutive);
                continue;
            }
            consecutive = 0;
        }
        return maxconsecutive;
    }
}